test/controllers/AccountingPeriodControllerSpec.scala

```diff
@@ -42,11 +43,27 @@ class AccountingPeriodControllerSpec extends SpecBase with MockitoSugar {
   val formProvider = new AccountingPeriodFormProvider()
   private def form = formProvider()
 
+  lazy val completedUserAnswers = UserAnswers(
```

Comment:
Minor: Can use .set method instead of parsing Json this way

Reply:
.set is a bit awkward to use here, since I can only set individual page data one at the time. Is there a way to set the complete form data of all pages all at once?

Reply:
Unfortunately no, however I use for comprehension and its slightly better

```
(for {
            u1 <- UserAnswers(userAnswersId).set(AccountingPeriodPage, AccountingPeriodForm(epoch,Some(date))
            u2 <- u1.set(TaxableProfitPage, 1)
            u3 <- u2.set(AssociatedCompaniesPage, AssociatedCompaniesForm())
            u4 <- u3.set(DistributionsIncludedPage, DistributionsIncludedForm(DistributionsIncluded.Yes, Some(1)))
          } yield u4)
```